plumber/cli_server.go

```diff
@@ -74,6 +82,8 @@ func (p *Plumber) RunServer() error {
 		p.log.Error(errors.Wrap(err, "failed to relaunch tunnels"))
 	}
 
+	go p.runRemoteControl()
```

Comment:
Should remote control be elective? It can be enabled by default but should be possible to disable.